CHANGES.md

```diff
@@ -10,6 +10,15 @@
 
 ### :sparkles: New features
 
+- **Replace Draft.js completely with a custom editor** [Taiga #7706](https://tree.taiga.io/project/penpot/us/7706)
+  
+  This refactor adds better IME support, more performant text editing 
+  experience and a better clipboard support while keeping full 
+  retrocompatibility with previous editor. 
+
+  To enable it you need to modify `config.js` and add the following 
+  flag to `penpotFlags`: `enable-feature-editor-v2`.
```

Comment:
I think this sentence should be like:

>> You can enable it with the `enable-feature-editor-v2` configuration flag.

It is not a good idea to specify the exact implementation detail, because the end users never touches the config.js file. The default installation method is using docker, and all flags are set in an uniform manner using environment vars.